governator-core/src/main/java/com/netflix/governator/internal/PreDestroyLifecycleActions.java

```diff
@@ -12,15 +17,18 @@
  * 
  * @author elandau
  */
-final class PreDestroyLifecycleActions extends AbstractLifecycleFeature {
+public final class PreDestroyLifecycleActions extends AbstractLifecycleFeature {
```

Comment:
I'd prefer the package private class instead of the public class behind an "internal" namespace if possible.